jekyll-docs/install.md

```diff
@@ -6,23 +6,6 @@ The installation has two parts – installation of the vSphere Installation Bund
 
 ## Installation on ESXi
 
-[VIB](http://blogs.vmware.com/vsphere/2011/09/whats-in-a-vib.html) and [Offline Depot](https://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html?resultof=%2522%256f%2566%2566%256c%2569%256e%2565%2522%2520%2522%256f%2566%2566%256c%2569%256e%2522%2520%2522%2564%2565%2570%256f%2574%2522%2520) are the packages built to install the backend for the service on ESX. The backend can be installed using esxcli or vmware tools such as [VUM](http://pubs.vmware.com/vsphere-60/topic/com.vmware.ICbase/PDF/vsphere-update-manager-601-install-administration-guide.pdf)
```

Comment:
@shuklanirdesh82 in one of the comments I was asked to remove all the references to VUM. [comment](https://github.com/vmware/docker-volume-vsphere/pull/1150#pullrequestreview-31465861)